scss/boosted-reboot.scss

```diff
@@ -3,6 +3,7 @@
 
 @import "functions";
 @import "variables";
+@import "variables-dark";
```

Comment:
Same

Reply:
It's weird but I think we don't have a choice contrary to `boosted/bootstrap-grid.scss` for now since reboot uses CSS vars or Sass vars defined in this `_variables-dark.scss`.